tools/azure-sdk-tools/ci_tools/functions.py

```diff
@@ -89,6 +89,9 @@ def filter_for_compatibility(package_set: List[str]) -> List[str]:
         if running_major_version in spec_set:
             collected_packages.append(pkg)
 
+    logging.debug("Target packages after applying compatibility filter: {}".format(collected_packages))
+    logging.debug("Package(s) omitted by business filter: {}".format(generate_difference(package_set, collected_packages)))
```

Comment:
I'm assuming these should be consistent -- correct me if I'm wrong
```suggestion
    logging.debug("Package(s) omitted by compatibility filter: {}".format(generate_difference(package_set, collected_packages)))
```

Reply:
These should absolutely be consistent thanks for the catch 👍